src/S2S/Request/Details/ExternalReferences.php

```diff
@@ -81,6 +83,21 @@ public function setAffiliateId(?string $affiliateId): void
         $this->affiliateId = $affiliateId;
     }
 
+    /**
+     * @return int|null
+     */
+    public function getInitialTransactionId(): ?int
```

Comment:
This doesn't reflect the actual implementation. The UnscheduledValidator expects the initial tranasction id in the unscheduled object, like this:
```
"unscheduled" : {
    "initialTransactionId": 123123123
}
```